Controller/DbControllers/PlaylistController.cs

```diff
@@ -76,9 +75,9 @@ public virtual void DeletePlaylistOnDateStamp()
          *
          * @returns List<Playlist> : a list of maximum 8 playlist based on the searchTerms
          */
-        public virtual List<Playlist> SearchPlayListOnString(List<string> searchTerms, int userId)
+        public List<Playlist> SearchPlayListOnString(List<string> searchTerms, User user)
         {
-            return GetActivePlaylists()
+            return GetActivePlaylists(user)
```

Comment:
?? de bedoeling was dat je alle playlisten kon zien, ook van andere users, zelf snap ik niet waarom er een userid in stond